ibm/conns/config.go

```diff
@@ -3266,23 +3269,24 @@ func (c *Config) ClientSession() (interface{}, error) {
 	if fileMap != nil && c.Visibility != "public-and-private" {
 		mqCloudURL = fileFallBack(fileMap, c.Visibility, "IBMCLOUD_MQCLOUD_CONFIG_ENDPOINT", c.Region, mqCloudURL)
 	}
-
+	accept_language := os.Getenv("IBMCLOUD_MQCLOUD_ACCEPT_LANGUAGE")
```

Comment:
what is this parameter used for?
If user doesn't set this ENV what is the default value?

Reply:
This is the env variable used to mention the language of the response message in the response object.

And if the user doesn't set this env variable, it will default to "" empty string value, which then the server will take it as "en" - English language.